### PR TITLE
Video editor merged

### PR DESCRIPTION
**Pull request by KUP9752**
Merged all sides of the video editor:
- The UI
- Editor functionality (timeline, export...)

### CODE REVIEW

File: desktop/electron/main.ts [hunk 1]
```diff
@@ -12,100 +17,121 @@ import { extractTextFromPDF, getScript, getTopics, setTopic} from './reportProce
 // │ │ ├── main.js
 // │ │ └── preload.js
 // │
-process.env.DIST = path.join(__dirname, '../dist')
-process.env.VITE_PUBLIC = app?.isPackaged ? process.env.DIST : path.join(process.env.DIST, '../public')
+process.env.DIST = path.join(__dirname, "../dist");
+process.env.VITE_PUBLIC = app.isPackaged
```
Comment (acowusu) [inline desktop/electron/main.ts:21]:
```suggestion
process.env.VITE_PUBLIC = app?.isPackaged
```

File: desktop/electron/main.ts
Comment (acowusu) [inline desktop/electron/main.ts:1]: I don't think any of the changes in this file are intended

File: desktop/electron/main.ts (hunk 1 above)
Comment (KUP9752) [reply desktop/electron/main.ts]: Yep fixed

File: desktop/electron/main.ts
Comment (KUP9752) [reply desktop/electron/main.ts:1]: yep, mostly my reformatter when I was messing with the app. Should all be removed now

File: desktop/package.json [hunk 2]
```diff
@@ -69,7 +71,7 @@
     "react-hook-form": "^7.49.3",
     "react-router-dom": "^6.21.2",
     "sonner": "^1.3.1",
-    "sort-by": "^1.2.0",
+    "sort-by": "^0.0.2",
```
Comment (acowusu) [inline desktop/package.json:74]: why are you downgrading this package

File: desktop/electron/main.ts
Comment (acowusu) [inline desktop/electron/main.ts:1]: In general can you not reformat entire files as it makes it impossible to see what has changed between commits.

You should be able to cofigure this in your ide settings
. Can you revert
```
git checkout main -- desktop/electron/main.ts
```
and then commit

File: desktop/src/components/timeline-components/timeFrame.tsx
Comment (acowusu) [inline desktop/src/components/timeline-components/timeFrame.tsx:1]: can u add a unit test for this

File: desktop/src/index.css [hunk 3]
```diff
@@ -1,76 +1,92 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
- 
+
```
Comment (acowusu) [inline desktop/src/index.css:4]: what is this for

File: desktop/src/pages/mediaFiles.tsx
Comment (acowusu) [inline desktop/src/pages/mediaFiles.tsx:1]: We should not have api keys directly in our source code!


At the very least we should load these from an `.env` file

see https://vitejs.dev/guide/env-and-mode

File: desktop/src/pages/mediaFiles.tsx [hunk 4]
```diff
@@ -55,36 +89,81 @@ export const Media: React.FC = () => {
...
     </div>
   );
 };
+
+{
+  /* <label htmlFor="querySelector">Select a category: </label>
+        <select
+          id="querySelector"
+          value={selectedQuery}
+          onChange={handleQueryChange}
```
Comment (acowusu) [inline desktop/src/pages/mediaFiles.tsx:160]: Is there a reason for this commented out code

File: frontend/main.js [hunk 5]
```diff
@@ -1,40 +1,42 @@
-const { app, BrowserWindow } = require('electron');
-const next = require('next');
+const { app, BrowserWindow } = require("electron");
+const next = require("next");
```
Comment (acowusu) [inline frontend/main.js:3]: Are these changes intended i thought we were nolonger using frontend

**Comment by acowusu** [comment]
Closing this as stale
